workflow/artifacts/raw/raw.go

```diff
@@ -0,0 +1,26 @@
+package raw
+
+import (
+	"github.com/argoproj/argo/errors"
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"os"
```

Comment:
This looks like it will fail gofmt.

Reply:
make lint didn't complain..will check again